dstk-infra/apollo/src/graphql/model-version/modelVersion.ts

```diff
@@ -1,27 +1,32 @@
-import { objectType } from 'nexus';
+import { builder } from '../../builder.js';
 import { MLModel, ObjectionMLModel } from '../model/model.js';
 import { Model } from 'objection';
 import { ObjectionStorageProvider } from '../storage-provider/storageProvider.js';
 
-export const MLModelVersion = objectType({
-    name: 'MLModelVersion',
-    definition(t) {
-        t.id('modelVersionId');
-        t.field('modelId', {
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const MLModelVersion = builder.objectRef<any>('MLModelVersion').implement({
```

Comment:
I haven't found a better way than this, although I don't love the explicit `any`. Can circle back if this pattern starts to bug us enough